src/adapters/stargate/products/pool/stargatePoolAdapter.ts

```diff
@@ -216,76 +214,84 @@ export class StargatePoolAdapter implements IProtocolAdapter {
       this.getPositions({
         userAddress,
         blockNumber: toBlock,
-      }).then((result) => formatToMap(result)),
-
+      }).then((positions) => formatProtocolTokenArrayToMap(positions)),
       this.getPositions({
         userAddress,
         blockNumber: fromBlock,
-      }).then((result) => formatToMap(result)),
+      }).then((positions) => formatProtocolTokenArrayToMap(positions)),
```

Comment:
![image](https://github.com/consensys-vertical-apps/mmi-defi-adapters/assets/32621022/068de6a9-c2f3-40d7-831e-971553e863b7)

nit picking but might be a nice iteration

Reply:
Done. I'm always weary of using that for readability, but in this case it's pretty obvious.